compose/utils.py

```diff
@@ -49,6 +49,9 @@ def split_buffer(stream, splitter=None, decoder=lambda a: a):
     splitter = splitter or line_splitter
     buffered = six.text_type('')
 
+    if type(stream) is bytes:
+        stream = stream.decode("utf-8")
```

Comment:
In python3, with option stream as False, `container.logs` returns an object of type `<class 'bytes'>`, in python2, the same api returns `<type 'str'>`. I converted bytes to string to patch it, but maybe is a bug of dockerpy ?